src/validator.ts

```diff
@@ -1,27 +1,49 @@
-import { FieldValidationFunctionSync } from '@lemoncode/fonk';
-import { CustomValidatorArgs } from './validator.model';
-import { isDefined, buildCustomMessage } from './validator.business';
+import {
+  FieldValidationFunctionSync,
+  parseMessageWithCustomArgs,
+} from '@lemoncode/fonk';
 
-// TODO: Add validator type
-const VALIDATOR_TYPE = '';
+const VALIDATOR_TYPE = 'RANGE_NUMBER';
 
-// TODO: Add default message
-let defaultMessage = '';
+let defaultMessage =
+  'The value must be between {{min.value}} and {{max.value}}';
 export const setErrorMessage = message => (defaultMessage = message);
 
+const isDefined = value => value !== void 0 && value !== null && value !== '';
+
+const validateType = value => typeof value === 'number';
+
+const validate = (value, min: Limit, max: Limit) =>
+  (min.inclusive ? value >= min.value : value > min.value) &&
+  (max.inclusive ? value <= max.value : value < max.value);
+
+interface CustomValidatorArgs {
+  min: Limit;
+  max: Limit;
+}
+
+interface Limit {
+  value: number;
+  inclusive: boolean;
+}
+
 export const validator: FieldValidationFunctionSync<
   CustomValidatorArgs
 > = fieldValidatorArgs => {
-  const { value, message = defaultMessage } = fieldValidatorArgs;
+  const { value, message = defaultMessage, customArgs } = fieldValidatorArgs;
 
-  // TODO: Add validator
-  const succeeded = !isDefined(value) || ...;
+  const succeeded =
+    !isDefined(value) ||
+    (validateType(value) && validate(value, customArgs.min, customArgs.max));
 
   return {
     succeeded,
     message: succeeded
       ? ''
-      : buildCustomMessage((message as string) || defaultMessage, args),
+      : parseMessageWithCustomArgs(
+          (message as string) || defaultMessage,
```

Comment:
Why is it necessary the `|| defaultMessage` if we are using it on line 33?

Reply:
It is not necessary. Updated.